### PR TITLE
CodeBlock, Search Text , etc localization

### PR DESCRIPTION
**Pull request by zora-wang**
@sharadkjaiswal 
 Related problem is in : http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6320#

### CODE REVIEW

**Comment by sharadkjaiswal** [comment]
LGTM, @aparajit-pratap PTAL

File: src/DynamoCoreWpf/Views/CodeBlocks/CodeBlockEditor.xaml.cs [hunk 1]
```diff
@@ -38,11 +38,13 @@ public partial class CodeBlockEditor : UserControl
         public CodeBlockEditor()
         {
             InitializeComponent();
+            WatermarkLabel.Text = Properties.Resources.WatermarkLabelText;
         }
 
         public CodeBlockEditor(NodeView nodeView)
         {
             InitializeComponent();
+            WatermarkLabel.Text = Properties.Resources.WatermarkLabelText;
```
Comment (aparajit-pratap) [inline src/DynamoCoreWpf/Views/CodeBlocks/CodeBlockEditor.xaml.cs]: Could you just make this constructor call the other one without arguments and just initialize the `WatermarkLabel` in the first constructor?

**Comment by aparajit-pratap** [comment]
@zora-wang other than the comment above please resolve merge conflicts and merge. Thanks.

File: src/Engine/ProtoCore/Parser/Parser.cs [hunk 2]
```diff
@@ -4510,139 +4510,139 @@ public class Errors {
...
+			case 123: s = Properties.Resources.InvalidImperativeUnaryexpr; break;
+			case 124: s = Properties.Resources.InvalidImperativeUnaryop; break;
+			case 125: s = Properties.Resources.InvalidImperativeFactor; break;
+			case 126: s = Properties.Resources.InvalidImperativeLogicallop; break;
+			case 127: s = Properties.Resources.InvalidImperativeRelop; break;
+			case 128: s = Properties.Resources.InvalidImperativeAddop; break;
+			case 129: s = Properties.Resources.InvalidImperativeMulop; break;
+			case 130: s = Properties.Resources.InvalidImperativeBitop; break;
+			case 131: s = Properties.Resources.InvalidImperativeNum; break;
+			case 132: s = Properties.Resources.InvalidImperativePostFixOp; break;
```
Comment (aparajit-pratap) [inline src/Engine/ProtoCore/Parser/Parser.cs]: @junmendoza do these changes go directly in `Parser.cs` or in one of the `.atg` files?

Comment (aparajit-pratap) [inline src/Engine/ProtoCore/Parser/Parser.cs]: @zora-wang I checked this with @junmendoza . You cannot change `Parser.cs` or the ATG files in this case as these error strings are auto-generated by coco. You need to map these error strings to the properties in `Resources` class and then replace the parser generated error message with the corresponding string resource. I think you can do this in `BuildStatus::LogSyntaxError`. For example, say you have the parser error string = "openbracket expected", create a string property of the same or equivalent name like "openbracket_expected" and then localize the actual error string in this property. Let me know if you want to discuss this further. Please revert `Parser.cs` for now.

**Pull request closed without merging**